### DISCUSSION TITLE
Question about Molecular Surface Area on MolCalc

### DISCUSSION THREAD

**Discussion by ashnanirula** [jensengroup/molcalc#45] [Q&A] [answered]
Hi @charnley and @vladislavivanistsev!

I'm a chemistry student from Toronto, Canada, and am a huge fan of this tool - MolCalc! I think it is clean, easy-to-use and it has definitely taught me a lot about molecules. I don't have an actual issue for you to fix, but instead two questions to further my knowledge:

1) what type of surface area is being calculated in the value "Surface Area"? Is it van der Waals surface area, or solvent accessible surface area? 
2) How is "Total Solvation Energy" being calculated?

Thanks for developing this awesome visualizer, and let me know!
Ashna

**Comment by charnley** [comment, marked as answer]
Hi @ashnanirula ,

1. It is the solvent accessible surface area with PCM
2. That is a longer answer, but using PM3/PCM in GAMESS

- https://github.com/jensengroup/molcalc/blob/master/molcalc_lib/gamess_calculations.py#L68
- https://www.msg.chem.iastate.edu/gamess/
- https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0067725

I would recommend the book or lectures by Jan Jensen to get into computational chemistry

- Molecular Modeling Basics by Jan H. Jensen
- https://www.youtube.com/user/MolModBasics

**Reply by ashnanirula** [reply]
@charnley, thanks so much for your response - it helped a lot!
